server/rosetta/types.go

```diff
@@ -1,41 +1,92 @@
 package rosetta
 
 import (
-	"github.com/coinbase/rosetta-sdk-go/types"
-
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	"crypto/sha256"
 )
 
 // statuses
 const (
-	StatusSuccess  = "Success"
-	StatusReverted = "Reverted"
-	StageSynced    = "synced"
-	StageSyncing   = "syncing"
+	StatusTxSuccess   = "Success"
+	StatusTxReverted  = "Reverted"
+	StatusPeerSynced  = "synced"
+	StatusPeerSyncing = "syncing"
 )
-
-// misc
 const (
-	Log = "log"
+	DeliverTxSize           = sha256.Size
+	BeginEndBlockTxSize     = DeliverTxSize + 1
+	EndBlockHashStart       = 0x0
+	BeginBlockHashStart     = 0x1
+	BurnerAddressIdentifier = "burner"
```

Comment:
can you add relevant comments on these constants? I don't know what they mean or how they are used

Reply:
there's a whole paragraph explaining this in the code comments, but i guess i will replicate it

Reply:
all the rationale behind these constants is documented extensively in other parts of the code, i didn't feel like replicating documentation